src/vs/editor/contrib/inlineCompletions/browser/structuredLogger.ts

```diff
@@ -53,7 +53,7 @@ interface IFetchResult {
  * The sourceLabel must not contain '@'!
 */
 export function formatRecordableLogEntry<T extends IRecordableLogEntry>(entry: T): string {
-	return entry.sourceId + ' @@ ' + JSON.stringify({ ...entry, sourceId: undefined });
+	return entry.sourceId + ' @@ ' + JSON.stringify({ ...entry, modelUri: (entry as any).modelUri?.toString(), sourceId: undefined });
```

Comment:
Unsafe? Could this get mangled in product build?

Reply:
Thanks for paying attention to code smells :)
But I think it is fine here, this is debug logging only. And no, it cannot get mangled, as this is a json object (which would break all kinds of interop, if json object properties would get mangled).